vars/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+# List of unattended-upgrades options that need to be erased before set by this role
+# See https://github.com/hifis-net/ansible-role-unattended-upgrades/issues/94
+__clear_options:
```

Comment:
How did you come up with this list of options? Is there some link with the available options somewhere?

Also, how can we make sure to not miss the addition of newly added variables?

Reply:
These were my sources:
* [https://github.com/mvo5/unattended-upgrades#supported-options-reference](https://github.com/mvo5/unattended-upgrades#supported-options-reference)
* [`50unattended-upgrades.Ubuntu`](https://github.com/mvo5/unattended-upgrades/blob/master/data/50unattended-upgrades.Ubuntu)
* [`50unattended-upgrades.Debian`](https://github.com/mvo5/unattended-upgrades/blob/master/data/50unattended-upgrades.Debian)

> Also, how can we make sure to not miss the addition of newly added variables?

Well, check the new unattended-upgrades releases regularly.

Reply:
Thanks :+1: